docs/QUICKSTART.md

```diff
@@ -0,0 +1,46 @@
+# Quick Start for KeePassXC
+
+This procedure gets KeePassXC running on your computer with browser integration, 
+using the pre-built binaries available for [download](https://keepassxc.org/download) 
+on the [KeePassXC site](https://keepassxc.org).
+
+**TL;DR** KeePassXC saves your passwords securely.
+When you double-click a URL in KeePassXC, it launches your default browser to that URL.
+With Browser Integration configured, KeePassXC automatically enters 
+username/password credentials into web page fields.
+
+## Installing and Starting KeePassXC
+
+* [Download the native installer](https://keepassxc.org/download) and install 
+KeePassXC for your  Windows, macOS, or Linux computer in the usual way for your platform.
+* Open the KeePassXC application.
+* Create a new database and give it a Master Key that's used to unlock the database file. 
+This database holds entries (usernames, passwords, account numbers, notes) 
+for all your websites, programs, etc.
+* Create a few entries - enter the username, password, URL, and optionally notes about the entry.
+* KeePassXC securely stores those entries in the database.
+
+
+## To set up Browser Integration with KeePassXC:
+
+* *Within KeePassXC*, go to **Tools->Settings** (on macOS, go to **KeePassXC->Preferences**.) 
+* In **Browser Integration**, check `Enable KeePassHTTP server`
+Leave the other options at their defaults.
+* *In your default web browser,* install the KeePassHTTP-Connector extension/add-on. Instructions for [[Firefox]](https://addons.mozilla.org/en-US/firefox/addon/keepasshttp-connector/?src=api) or [[Chrome]](https://chrome.google.com/webstore/detail/keepasshttp-connector/dafgdjggglmmknipkhngniifhplpcldb?utm_source=chrome-app-launcher-info-dialog) 
+* Click the KeePassXC icon in the upper-right corner. You'll see the dialog below. 
+* Click the blue Connect button to make the browser extension connect to the KeePassXC application. 
+<img src="./KeePassXC-Connect.png" height="200" alt="KeePassXC Connect dialog">
+* *Switch back to KeePassXC.* You'll see a dialog (below) indicating that a request to connect has arrived. 
+* Give the connection a name (perhaps `Keepass-Browsername`, any unique name will suffice) and click OK to accept it.
+* This one-time operation connects KeePassXC and your browser.
+<img src="./KeePassXC-Accept-Button.png" height="200" alt="KeePassXC accept connection dialog">
+
+## To use Browser Integration
+
+* *Within the KeePassXC application,* double-click the URL of an entry,
+or select it and type Ctl-U (Cmd-U on macOS).
+* Your browser opens to that URL.
+* If there are Username/Password fields on that page, you will see the dialog below.
+Click "Allow" to confirm the KeePassXC access the credentials and to auto-fill the fields. 
```

Comment:
"Click "Allow" to confirm that KeePassXC may access the credentials to auto-fill the fields"